CMakeLists.txt

```diff
@@ -1,23 +1,40 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.14)
 
 include(pico_sdk_import.cmake)
 
 project(picoprobe)
 
 pico_sdk_init()
 
+# Keep at 0 for Raspberry Pi Pico
+# Set this to 1 for QTPy with WS2812
+# Set this to 2 for QTPy without WS2812
+set(USE_QTPY 0)
```

Comment:
Does `cmake` also allow you to set and compare strings? Having a `USE_QTPY` variable set to different numbers feels a bit arbitrary, and would obviously be hard to extend if somebody wanted to add support for a different RP2040 board?

Reply:
Done

Reply:
IMHO that looks **much** better now :+1: